src/widgets.js

```diff
@@ -284,7 +284,9 @@ define([
                             super_setValue(val);
                             widget.handleChange();
                         }
-                    }
+                    },
+                    form: mug.form,
+                    questionType: options.path,
```

Comment:
Oops. that snuck in from some other work I was doing on case management. form is used here: https://github.com/dimagi/Vellum/pull/438/files#diff-3514e86c408245666c24881d2a721031R84